### PR TITLE
PMM-9403 Add extra label for replicaset metrics

### PR DESCRIPTION
**Pull request by BupycHuk**
[PMM-9403](https://jira.percona.com/browse/PMM-9403) (optional, if ticket reported)

- [ ] Links to related pull requests (optional).
[FB](https://github.com/Percona-Lab/pmm-submodules/pull/3566)

---

Below we provide a basic checklist of things that would make it a good PR:
- Make sure to sign the CLA (Contributor License Agreement).
- Make sure all tests pass.
- Keep current with the target branch and fix conflicts if necessary.
- Update jira ticket description if necessary.
- Attach screenshots and/or console output to the jira ticket to confirm new behavior, if applicable.
- Leave notes to the reviewers if you need to focus their attention on something specific.

Once all checks pass and the code is ready for review, please add `pmm-review-exporters` team as the reviewer. That would assign people from the review team automatically. Report any issues on our [Forum](https://forums.percona.com).

[PMM-9403]: https://perconadev.atlassian.net/browse/PMM-9403?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: exporter/v1_compatibility.go [hunk 1]
```diff
@@ -999,8 +999,8 @@ func oplogStatus(ctx context.Context, client *mongo.Client) ([]prometheus.Metric
 	return []prometheus.Metric{headMetric, tailMetric}, nil
 }
 
-func replSetMetrics(m bson.M) []prometheus.Metric {
-	replSetGetStatus, ok := m["replSetGetStatus"].(bson.M)
+func replSetMetrics(d bson.M) []prometheus.Metric {
```
Comment (JiriCtvrtka) [inline exporter/v1_compatibility.go:1002]: Just out of curiosity. Why "d"? Since it is bson.M not bson.D.

**Review by JiriCtvrtka — APPROVED** [review]

File: exporter/v1_compatibility.go (hunk 1 above)
Comment (JiriCtvrtka) [reply exporter/v1_compatibility.go:1002]: Got it. Thanks

**Review by ademidoff — APPROVED** [review]
@BupycHuk  Unrelated, but still: can we remove this [line](https://github.com/percona/mongodb_exporter/blob/main/tools/go.mod#L5) before merging?

**Comment by BupycHuk** [comment]
@ademidoff let's do it in a separate PR

**Pull request merged**